{"name": "Проверка отказа в авторизации из-за незаполненного логина", "status": "passed", "start": 1722200242190, "stop": 1722200242272, "uuid": "eda6d089-5b28-4faf-9333-c4f19af9a5a3", "historyId": "5454b7e93028321e089ae8cd5709a2ac", "testCaseId": "5454b7e93028321e089ae8cd5709a2ac", "fullName": "tests.test_login_courier.TestLoginCourier#test_not_authorization_without_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DTALS"}, {"name": "thread", "value": "12476-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}